pom.xml

```diff
@@ -11,9 +11,9 @@
     <description>Alcor Controller Project</description>
 
     <properties>
-        <java.version>11</java.version>
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
+        <java.version>1.8</java.version>
```

Comment:
Not sure why we make the main JDK version here. Is it a merge issue?

Reply:
sorry, It is a merge issue... I fixed it